npm install issue with last xcode

looks like the recent xcode upgrade broke libxmljs :

```sh
3 warnings generated.
  CC(target) Release/obj.target/libxml/vendor/libxml/xpointer.o
  LIBTOOL-STATIC Release/xml.a
  CXX(target) Release/obj.target/xmljs/src/libxmljs.o
In file included from ../src/libxmljs.cc:7:
In file included from ../src/libxmljs.h:7:
In file included from ../node_modules/nan/nan.h:63:
../node_modules/nan/nan_new.h:207:43: error: inline declaration of 'NanNew' follows non-inline definition
template <typename T> inline v8::Local<T> NanNew(v8::Handle<T> h);
                                          ^
../node_modules/nan/nan_implementation_pre_12_inl.h:210:1: note: previous definition is here
NanNew(v8::Handle<T> h) {
^
In file included from ../src/libxmljs.cc:7:
In file included from ../src/libxmljs.h:7:
In file included from ../node_modules/nan/nan.h:63:
../node_modules/nan/nan_new.h:208:43: error: inline declaration of 'NanNew' follows non-inline definition
template <typename T> inline v8::Local<T> NanNew(v8::Persistent<T> const& p);
                                          ^
../node_modules/nan/nan_implementation_pre_12_inl.h:216:1: note: previous definition is here
NanNew(v8::Persistent<T> const& p) {
^
2 errors generated.
make: *** [Release/obj.target/xmljs/src/libxmljs.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:810:12)
gyp ERR! System Darwin 14.1.0
gyp ERR! command "node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/juju/Documents/projects/xxxx/node_modules/plist-native/node_modules/libxmljs
gyp ERR! node -v v0.10.32
gyp ERR! node-gyp -v v1.0.2
gyp ERR! not ok
npm ERR! Darwin 14.1.0
npm ERR! argv "node" "/usr/local/bin/npm" "i"
npm ERR! node v0.10.32
npm ERR! npm  v2.1.2
npm ERR! code ELIFECYCLE

npm ERR! libxmljs@0.13.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the libxmljs@0.13.0 install script.
npm ERR! This is most likely a problem with the libxmljs package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls libxmljs
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/juju/Documents/projects/xxxx/npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -17,7 +17,7 @@
     "xml"
   ],
   "dependencies": {
-    "libxmljs": "^0.13.0"
+    "libxmljs": "^0.14.0"
   },
   "devDependencies": {
     "nodeunit": "0.9.1"

